packages/machine/src/models/app-instance.ts

```diff
@@ -67,7 +67,7 @@ export class AppInstance {
     isVirtualApp: boolean,
     appSeqNo: number,
     rootNonceValue: number,
-    latestState: object,
+    latestState: any,
```

Comment:
why `object` to `any`?

Reply:
Actually this seems wrong, it should be the same as `AppState`: https://github.com/counterfactual/monorepo/blob/master/packages/machine/src/protocol-types-tbd.ts#L16